modules/bitgo/test/v2/unit/coins/utxo/recovery/crossChainRecovery.ts

```diff
@@ -31,6 +31,7 @@ import {
   supportedCrossChainRecoveries,
 } from '@bitgo/abstract-utxo';
 import * as sinon from 'sinon';
+import { Bch } from 'modules/bitgo/src/v2/coins';
```

Comment:
Is this correct place to import this from?

![image](https://user-images.githubusercontent.com/24881706/217296942-d42e0d1b-13ee-41ae-b24c-bd038278daa5.png)